hackerrank/1-week-prep/day-4/mock-test-4.py

```diff
@@ -0,0 +1,16 @@
+def truckTour(petrol_pumps):
+    for i in range(len(petrol_pumps)):
+        if can_complete_circle(petrol_pumps, i):
+            return i
+    return None
+
+def can_complete_circle(petrol_pumps, current, cummulated_petrol=0, visited_pumps=0):
```

Comment:
Since you're only calling `can_complete_circle` in line 3, `cumulated_petrol` and `visited_pumps` don't need to be function parameters.